src/plans/details/tabs/Mappings/components/PlanMappingsActionsBar/PlanMappingsActionsBar.tsx

```diff
@@ -43,7 +43,9 @@ const PlanMappingsActionsBar: FC<PlanMappingsActionsBarProps> = ({
         <FlexItem>
           <Button
             variant={ButtonVariant.secondary}
-            onClick={reset}
+            onClick={() => {
+              reset();
+            }}
```

Comment:
why didn't it work with original code?

Reply:
since onClick is essentially have event argument, then it translates to: onClick={reset} === (event) => { reset{event); } which give a wrong value for the reset function and caused the bug

Reply:
Right...the prototype for reset  is set in props as
`reset: () => void;:`
while actually it should be declared as
`reset: (preserveUpdated?: boolean) => void;`
And that caused the mistake I guess....